lib/bank_account/checking_account.rb

```diff
@@ -0,0 +1,64 @@
+require "pry"
+
+module BankAccount
+
+  class CheckingAccount < Account
+
+    # This is unique to the checking account
+    attr_accessor :free_checks
+
+    def initialize(id, initial_balance)
+      super
+      @free_checks = 3 #Per month
+    end
+
+    # Get monies
+    def withdraw(amount)
+
+      slight_burglary = 1 #That yucky fee
+      total_withdraw = amount + slight_burglary
```

Comment:
This is a nice use of a local variable since this calculation is needed more than once